crates/bevy-inspector-egui/examples/quick/filter_query_inspector.rs

```diff
@@ -29,15 +29,15 @@ fn setup(
         ..default()
     });
     // light
-    commands.spawn(PointLightBundle {
+    /*commands.spawn(PointLightBundle {
```

Comment:
Accidentally left commented out?

Reply:
Not sure about this one (as I inherited from https://github.com/mvlabat/bevy_egui/pull/188 ). 

I uncommented it and ran `cargo test --all-features` and all `test result` came `ok`.